website/docusaurus.config.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
         alt: 'FlutterFire Logo',
         src: '/img/flutterfire_300x.png',
       },
-      links: [
+      items: [
```

Comment:
Can we leave this out of the PR? I need to fix the build versions for the website gen.